portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,14 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+/* Nav bar was eating the top at the beginning befor scrolling,
+so I added a space before that */
+body::before {
+    display: block;
+    content: '';
+    height: 50px;
```

Comment:
[2.2.1](https://google.github.io/styleguide/htmlcssguide.html#Indentation): Indent by 2 spaces at a time.

After you've set **Editor: Tab Size** to 2, I think you can fix this by selecting the entire document and outdenting with `Shift` + `Tab` or `Ctrl`/`Cmd` + `[`.